setup.py

```diff
@@ -3,10 +3,13 @@
 with open("README") as f:
     long_description = f.read()
 
-version = '1.18.0'
+
+def local_scheme(version):
+    """Skip the local version (eg. +xyz of 0.6.1.dev4+gdf99fe2)
+    to be able to upload to Test PyPI"""
+    return ""
```

Comment:
For `master`, the default version from setuptools_scm is something like `0.6.1.dev4+gdf99fe2`. PyPI doesn't like the `+gdf99fe2` part ("local version"), so this tells setuptools_scm to skip it